.github/workflows/build-and-test.yaml

```diff
@@ -10,6 +10,20 @@ jobs:
     name: Build and test
     runs-on: ubuntu-latest
 
+    services:
+      db:
+        image: postgres
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_USER: 'postgres'
+          POSTGRES_PASSWORD: '**your-postgres-pass-here**'
+          POSTGRES_DB: 'b2b_opportunities_test'
```

Comment:
I think we can change these values POSTGRES_USER , POSTGRES_PASSWORD to be env. variables, just like in the main DB

Reply:
I'm not sure the action will pass then, because github won't have a source to get this variables to initialize the database.

Reply:
There is a way to allow github actions to access the .env file from infisical.com, or we can just add the variables manually. Both ways should work, however the first option is probably the better approach as it will constantly pull those variables even if we change them or we add new ones.

Here's what I think needs to be added to the .github/workflows/build-and-test.yaml file:

```

- name: Install Infisical CLI
  run: |
    curl -1sLf 'https://dl.cloudsmith.io/public/infisical/infisical-cli/setup.deb.sh' | sudo -E bash
    sudo apt update && sudo apt install -y infisical

- name: Pull environment variables from Infisical
        env:
          INFISICAL_TOKEN: ${{ secrets.INFISICAL_TOKEN }}
        run: infisical pull --environment=development --output=.env
		
- name: Build
  run: source .env && ./gradlew build --no-daemon

- name: Test
  run: source .env && ./gradlew test --info --no-daemon

```

I've created the token, but I don't have access to the project settings and I can't set it up as a variable myself. We'll ask Emo to add it